### ISSUE TITLE
Make Adguard's Block Page, Settings Page run in correct windows mode that user is using.

### ISSUE THREAD

**Issue by Nekroz88** [AdguardTeam/AdguardBrowserExtension#2077]
- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Problem Description

It's good to see that Adguard is able to block some domains when I surf the internet.
But there is one annoying thing, is that _Adguard always runs within the main windows mode_.

For example, when I am browsing in browser's private/incognito mode, access a site and then Adguard blocks it.
However, Adguard pops up in a new main windows, not private windows that I am using, and I don't want that "Access Denied" (Blocked by Adguard) popped up page to go into my browsing history.
Meanwhile, Ublock Origin is able to run, popup within the private and main browser windows seperately.
The same goes for Adguard's Settings page as well, but it's lower priority, less annoying, at least for me.

Can you guys consider adding feature(s) to make Adguard's popup tabs/windows able to run in correct browsing mode that the user is using?

**Comment by maximtop** [comment]
We already have a similar issue here https://github.com/AdguardTeam/AdguardBrowserExtension/issues/2071

In order to make this possible we have to update the incognito field value https://developer.chrome.com/docs/extensions/mv3/manifest/incognito/#split

and test if won't break the usual work process of the extension

**Comment by Birbber** [comment]
Closing as a duplicate of [#2071](https://github.com/AdguardTeam/AdguardBrowserExtension/issues/2071).
